village/src/components/SmurfForm.js

```diff
@@ -11,15 +18,27 @@ class SmurfForm extends Component {
   }
 
   addSmurf = event => {
-    event.preventDefault();
+    //event.preventDefault();
     // add code to create the smurf using the api
-
-    this.setState({
-      name: '',
-      age: '',
-      height: ''
+    axios
+      .post('http://localhost:3333/smurfs', this.state)
+      .then(response => {
+        this.setState({
+          name: '',
+          age: '',
+          height: ''
     });
-  }
+  })}
+
+  handleUpdateSmurf = smurfId => {
```

Comment:
You don't actually send it a smurfID, which is why it isn't working. I assume this is probably where you were caught at the end of the day.